src/main/java/com/aws/greengrass/security/SecurityService.java

```diff
@@ -163,6 +165,61 @@ public KeyManager[] getKeyManagers(URI privateKeyUri, URI certificateUri)
         return provider.getKeyManagers(privateKeyUri, certificateUri);
     }
 
+    /**
+     * Get certificate chain using key alias.
+     *
+     * @param privateKeyUri  private key URI
+     * @param certificateUri certificate URI
+     * @return X509Certificate that corresponds to the given private key and certificate URIs
+     * @throws ServiceUnavailableException if crypto key provider service is unavailable
+     * @throws KeyLoadingException         if crypto key provider service fails to load key
+     */
+    public X509Certificate[] getCertificate(URI privateKeyUri, URI certificateUri)
```

Comment:
This api name is wrong. It isn't giving us the certificate, it is giving us the certificate chain. I don't believe that we do want (or even have) the chain. We only have a single certificate, so I'd just return that.